spec/read_models/caseworker_reports/link_to_temporal_streams_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe CaseworkerReports::LinkToTemporalStreams do
+  let(:event_store) { instance_double(RailsEventStore::Client) }
+  let(:event) { Reviewing::Completed.new(metadata: { timestamp: }) }
+  let(:timestamp) { Time.zone.local(2023, 8, 1, 22, 59) }
+
+  describe '#call' do
+    subject(:call) { described_class.new(event_store:).call(event) }
+
+    before do
+      allow(event_store).to receive(:link)
+      call
+    end
+
+    it 'links to the monthly stream based on the year and month of the year' do
+      stream_name = 'MonthlyCaseworker$2023-08'
+      expect(event_store).to have_received(:link).with([event.event_id], stream_name:)
+    end
+
+    it 'links to the weekly stream based on year and week of the year' do
```

Comment:
I was wondering about week boundaries as weeks can span into two years. For example, week 52 of 2023 might start in 2023 and end in 2024. 

Would this this mean that data related to 2024 is being stored under a 2023 stream?